lib/src/main/java/com/diffplug/spotless/FormatExceptionPolicy.java

```diff
@@ -17,10 +17,12 @@
 
 import java.io.Serializable;
 
+import javax.annotation.Nullable;
+
 /** A policy for handling exceptions in the format. */
 public interface FormatExceptionPolicy extends Serializable, NoLambda {
 	/** Called for every error in the formatter. */
-	void handleError(Throwable e, FormatterStep step, String relativePath);
+	public void handleError(@Nullable Throwable e, FormatterStep step, String relativePath);
 
```

Comment:
See the bottom of my response [here](https://github.com/diffplug/spotless/pull/101#issuecomment-294349655). :)